_observing-your-data/alerting/monitors.md

```diff
@@ -360,166 +55,80 @@ Variable | Data type | Description
 
-Q: Do I need to install the Notifications plugins if monitors can still use destinations?
-A: Yes. The fallback on destination is to prevent failures in sending messages if migration fails; however, the Notification plugin is what actually sends the message. Not having the Notification plugin installed will lead to the action failing.
+The Alerting plugin also creates a list of document findings that contains metadata about which document matches each query. Security analytics can use the document findings data to keep track of and analyze the query data separately from the alert processes.
 
+The Alerting API provides a _document-level monitor_ that programmatically accomplishes the same function as the _per document monitor_ in the OpenSearch Dashboards. See [Document-level monitors]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/api/#document-level-monitors) to learn more.
+{: .note}
 
----
+### Searching document findings
 
-## Work with alerts
+When a per document monitor executes a query that matches a document in an index, a finding is created. OpenSearch provides a findings index, `.opensearch-alerting-finding*`, that contains findings data for all per document monitor queries. You can search the findings index with the Alerting API search operation. See [Search for monitor findings]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/api/#search-for-monitor-findings) for more information.
 
-Alerts persist until you resolve the root cause and have the following states:
+The following metadata is provided for each document findings entry:
 
-State | Description
-:--- | :---
-Active | The alert is ongoing and unacknowledged. Alerts remain in this state until you acknowledge them, delete the trigger associated with the alert, or delete the monitor entirely.
-Acknowledged | Someone has acknowledged the alert, but not fixed the root cause.
-Completed | The alert is no longer ongoing. Alerts enter this state after the corresponding trigger evaluates to false.
-Error | An error occurred while executing the trigger---usually the result of a a bad trigger or destination.
-Deleted | Someone deleted the monitor or trigger associated with this alert while the alert was ongoing.
+* **Document** – The document ID and index name. For example: `Re5akdirhj3fl | test-logs-index`.
+* **Query** – The query name that matched the document.
+* **Time found** – The timestamp that indicates when the document was found during the runtime.
 
----
+While you can configure an alert notification for each finding, it is not recommended unless rules are well defined to prevent a large volume of findings in a high ingestion cluster.
```

Comment:
```suggestion
To prevent a large volume of findings in a high ingestion cluster, configuring alert notifications for each finding is not recommended unless rules are well defined.
{: .important}
```